pkg/blobcache/blobcache.go

```diff
@@ -403,7 +411,7 @@ func saveStream(wg *sync.WaitGroup, decompressReader io.ReadCloser, tempFile *os
 }
 
 func (s *blobCacheDestination) HasThreadSafePutBlob() bool {
-	return false
+	return true
```

Comment:
This needs to respect the `HasThreadSafeGetBlob` value of the underlying destination.

Reply:
@mtrmac You mean `HasThreadSafePutBlob()`, right?

Reply:
Yes, sorry, a copy&paste error.